src/librustc/arena.rs

```diff
@@ -216,6 +216,9 @@ arena_types!(declare_arena, [], 'tcx);
 
 arena_types!(impl_arena_allocatable, [], 'tcx);
 
+// Needed until #68057 makes it way into the bootrap compiler,
+// as librustc is affected by the bug that it fixes
+#[cfg_attr(not(bootstrap), marker)]
```

Comment:
It's possible that the new bootstrap compiler is here before this PR merges. If so, this will need to be updated.